src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -331,7 +332,11 @@ def _merge(self, dict_1, dict_2, identifier):
                 merged_dict[key] = list(merged_dict[key] + [value])
             elif isinstance(value, list):
                 cleaned_list = list(filter(None, chain(value, merged_dict[key])))
-                merged_dict[key] = list(set(cleaned_list))
+                if len(cleaned_list) > 0 and isinstance(cleaned_list[0], dict):  # make dicts hashable
+                    cleaned_list = [freeze(d) for d in cleaned_list]
+                    merged_dict[key] = [thaw(d) for d in list(set(cleaned_list))]
```

Comment:
I hope there must be the better way to produce a unique list of dictionary objects. This would cause Python to create new objects and therefore additional memory consumption.

This is a low priority but I wonder if you can write a helper method to produce a list of unique `dict` objects by iterating object instead of using `freeze` and `thaw`.